UITests/BookmarkSortTests.swift

```diff
@@ -61,6 +61,7 @@ class BookmarkSortTests: XCTestCase {
         app.dismissPopover(buttonIdentifier: "Hide")
         app.openBookmarksPanel()
         selectSortByName(mode: .panel)
+        app.openBookmarksPanel() // Here we do not open the panel, we close it by tapping the shortcut button again.
```

Comment:
I plan to refactor this method in the future; this method, in reality, taps the bookmarks panel button, and if it is not present, it will show it using a keyboard shortcut, and then tap it.

The `open` part is confusing because if you call this method and the panel is already opened, it will close it.

Reply:
We could consider renaming the method to something like `clickBookmarksShortcut` perhaps, but I wouldn't block the PR on it.